pytest_tests/steps/s3_gate_base.py

```diff
@@ -59,6 +60,38 @@ def s3_client(self, prepare_wallet_and_deposit, client_shell: Shell, request, ho
         TestS3GateBase.s3_client = client
         TestS3GateBase.wallet = wallet
 
+    @pytest.fixture
+    @allure.title("Create/delete bucket")
+    def bucket(self):
+        bucket = s3_gate_bucket.create_bucket_s3(self.s3_client)
+        yield bucket
+        self.delete_all_object_in_bucket(bucket)
+
+    @pytest.fixture
+    @allure.title("Create two buckets")
+    def create_buckets(self):
```

Comment:
create_bucket -> two_buckets. Since non-autouse fixture should represent a "thing", not an "action" :)

Reply:
done